Cargo.toml

```diff
@@ -266,6 +266,11 @@ name = "ring"
 libc = "0.2.20"
 untrusted = "0.3.2"
 
+[build-dependencies]
+gcc = "0.3"
+target_build_utils = "0.2"
```

Comment:
I filed #457 for removing the target_build_utils dependency after this PR is merged. The target_build_utils itself is about 10 additional transitive dependencies, which is too much long-term. It's OK for now, in order to make forward progress.